net/sslh/src/opnsense/service/conf/sslh.conf

```diff
@@ -0,0 +1,7 @@
+[plugin]
```

Comment:
looking at this file for the first time the idea is good, but I doubt softcoding these never-changing values provides much benefit.

Reply:
Understandable with a small plugin like this as an example, it could be used more extensively with larger plugins.

These settings can then be used throughout the plugin, in PHP( and thus volt) via parse_ini_file, and in python via configparse module (still exploring ideas on getting them into the Jinja template environment). With the outcome being code which is more portable, instead of having these values statically defined throughout the entire plugin. Hopefully it would be less work for someone else tomorrow.

I wasn't sure about the location of the conf file since that directory is actually for configd and the conf sits alongside `configd.conf`, but shouldn't clobber anything if it were to also be used by plugins too. Maybe another location is more suitable?

Reply:
I made two suggestions about the location elsewhere. As said before the main concern now is reading the file for read operations of the GUI... that could slow things down eventually if more plugins use this approach.